### DISCUSSION TITLE
The time is incorrect.

### DISCUSSION THREAD

**Discussion by zzzPitoNzzz** [ramensoftware/windhawk-mods#85] [Show and tell]
After midnight there should be zeros, not the digit 12. Everything is visible in the screenshot. One o'clock in the morning is already showing normally.
![времядатавпроге1](https://user-images.githubusercontent.com/114881552/212499616-0366853e-169d-4657-8209-661d12f1f244.jpg)

**Comment by m417z** [comment]
The time is formatted by Windows according to the formatting rules:
https://learn.microsoft.com/en-us/windows/win32/api/datetimeapi/nf-datetimeapi-gettimeformatex#remarks

Perhaps the mod is using:
> hh - Hours with leading zero for single-digit hours; 12-hour clock

and the left app is using:
> H - Hours with no leading zero for single-digit hours; 24-hour clock

You can configure the format in the mod's settings.

**Comment by zzzPitoNzzz** [comment]
Yes, thank you! I'll check tomorrow. I just noticed and wrote.15.01.2023, 01:25, "Michael Maltsev" ***@***.***>:
The time is formatted by Windows according to the formatting rules:
https://learn.microsoft.com/en-us/windows/win32/api/datetimeapi/nf-datetimeapi-gettimeformatex#remarks
Perhaps the mod is using:

hh - Hours with leading zero for single-digit hours; 12-hour clock
and the left app is using
H - Hours with no leading zero for single-digit hours; 24-hour clock

You can configure the format in the mod's settings.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
-- С уважением, Дементьев Олег

**Comment by zzzPitoNzzz** [comment]
Ок.01.04.2023, 01:07, "Michael Maltsev" ***@***.***>:
Closed #85 as resolved.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
-- С уважением, Дементьев Олег Александрович.https://www.facebook.com/oleg.dementyev https://vk.com/olegd2511https://twitter.com/Oleg_Dementev
